examples/04_dimension_reduction_and_performance.py

```diff
@@ -116,7 +116,8 @@ def wrapped_func(*args, **kwargs):
 # We can run a cross-validation
 from sklearn import linear_model, pipeline, model_selection
 
-log_reg = linear_model.LogisticRegression()
+# We add max_iter to avoid ConvergenceWarning
```

Comment:
```suggestion
# We specify max_iter to avoid convergence warnings
```
I'd rather write "convergence warnings" then "ConvergenceWarning" to facilitate the understanding by non technical users.